dlclibrary/dlcmodelzoo/modelzoo_download.py

```diff
@@ -20,10 +22,10 @@
     "mouse_pupil_vclose",
     "horse_sideview",
     "full_macaque",
-    "superanimal_topviewmouse",
-    "superanimal_quadruped",
-    "superanimal_quadruped_HRNetw32",
-    "superanimal_quadruped_ATP",
+    "superanimal_topviewmouse_dlcrnet",
+    "superanimal_quadruped_dlcrnet",
+    "superanimal_topviewmouse_hrnetw32",
+    "superanimal_quadruped_hrnetw32",
```

Comment:
here we do not have `superanimal_quadruped_dlcrnet` anymore, available superanimal models:

```
superanimal_topviewmouse (hrnet_w32) -- 'superanimal_topviewmouse_hrnetw32'
superanimal_quadruped (hrnet_w32) -- 'superanimal_quadruped_hrnetw32'
superanimal_topviewmouse (dlcrnet) -- 'superanimal_topviewmouse_dlcrnet'
```